src/RESTController.ts

```diff
@@ -332,3 +348,4 @@ const RESTController = {
 };
 
 module.exports = RESTController;
+export default RESTController;
```

Comment:
This red symbol means you are missing a new line at the end of the file

Reply:
Fixed

Reply:
There are a lot of new lines missing. We should switch from eslint to tsc (`npm run build:types`) in a separate PR.